### ISSUE TITLE
Open Collective Payment Method

### ISSUE THREAD

**Issue by piamancini** [opencollective/opencollective#919]
We are discussing our roadmap for next payment methods for Open Collective. Which one should we do first? Please click on your preferred option. 

[![](https://api.gh-polls.com/poll/01C6X2ER4JZ3EM9WB35334DX14/ACH)](https://api.gh-polls.com/poll/01C6X2ER4JZ3EM9WB35334DX14/ACH/vote)
[![](https://api.gh-polls.com/poll/01C6X2ER4JZ3EM9WB35334DX14/Alipay)](https://api.gh-polls.com/poll/01C6X2ER4JZ3EM9WB35334DX14/Alipay/vote)
[![](https://api.gh-polls.com/poll/01C6X2ER4JZ3EM9WB35334DX14/Android%20Pay)](https://api.gh-polls.com/poll/01C6X2ER4JZ3EM9WB35334DX14/Android%20Pay/vote)
[![](https://api.gh-polls.com/poll/01C6X2ER4JZ3EM9WB35334DX14/Apple%20Pay)](https://api.gh-polls.com/poll/01C6X2ER4JZ3EM9WB35334DX14/Apple%20Pay/vote)
[![](https://api.gh-polls.com/poll/01C6X2ER4JZ3EM9WB35334DX14/BanContact)](https://api.gh-polls.com/poll/01C6X2ER4JZ3EM9WB35334DX14/BanContact/vote)
[![](https://api.gh-polls.com/poll/01C6X2ER4JZ3EM9WB35334DX14/Bitcoin)](https://api.gh-polls.com/poll/01C6X2ER4JZ3EM9WB35334DX14/Bitcoin/vote)
[![](https://api.gh-polls.com/poll/01C6X2ER4JZ3EM9WB35334DX14/Ethereum)](https://api.gh-polls.com/poll/01C6X2ER4JZ3EM9WB35334DX14/Ethereum/vote)
[![](https://api.gh-polls.com/poll/01C6X2ER4JZ3EM9WB35334DX14/SEPA)](https://api.gh-polls.com/poll/01C6X2ER4JZ3EM9WB35334DX14/SEPA/vote)
[![](https://api.gh-polls.com/poll/01C6X2ER4JZ3EM9WB35334DX14/Paypal)](https://api.gh-polls.com/poll/01C6X2ER4JZ3EM9WB35334DX14/Paypal/vote)

**Comment by elopio** [comment]
Probably not a good candidate for the first iteration, but it would be great to support faircoin in the near future :)

**Comment by auvipy** [comment]
is visa/master card supported?

**Comment by EugeneHlushko** [comment]
Living outside of EU/US makes me suffer from numerous things i need to go through while getting any rewards out as we don't even have PayPal legalised in Ukraine. MC/VISA or direct invoicing via bank accs would save our days.

**Comment by piamancini** [comment]
@auvipy yes, Stripe supports credit cards already. These are alternative payment methods.

@EugeneHlushko we opened a transferwise account to explore alternative options to paypal for withdrawals. stay tuned to news on this. Thank you!

**Comment by guidojo** [comment]
Adding paypal would be great! In a lot of countries, most people don't have a credit card but they have their PayPal linked with their Bank account. This is the only reason I'm on Patreon iso OC

**Comment by westurner** [comment]
Coinbase and BitPay waive fees for 501c3 organizations:

https://developers.coinbase.com/docs/wallet/tutorials/donations

https://bitpay.com/docs/donations

https://bitcoin.tax does handle donations

**Comment by christophelassuyt** [comment]
- Coinbase and BitPay are good centralized solutions / companies to handle payments in ETH, BTC, LTC and BCH.
- I am working in [Request Network](https://request.network) team, we're creating a free decentralized payment requests network to get paid in any crypto-currency (including ETH, BTC, etc...). 
The library is here https://github.com/RequestNetwork/

**Comment by westurner** [comment]
- KYC: Know Your Customer
- AML: Anti-Money Laundering
- GDPR: EU General Data Protection Regulation
- PII: Personally Identifiable Information

- In order to get a charitable contribution tax deduction for a donation to a 501(c)3 nonprofit, donors need a tax receipt with their PII.
  - Storing PII in a blockchain may not (?) be GDPR-compliant; and may not be the privacy that a donor who just wants a receipt is looking for.
- https://www.irs.gov/taxtopics/tc506

  > You can only deduct charitable contributions if you itemize deductions
  > 
  > You must fill out Form 8283.pdf, Noncash Charitable Contributions, and attach it to your return, if your deduction for a noncash contribution is more than $500. If you claim a deduction for a contribution of noncash property worth $5,000 or less, you must fill out Form 8283, Section A. If you claim a deduction for a contribution of noncash property worth more than $5,000, you'll need a qualified appraisal of the noncash property and must fill out Form 8283, Section B. If you claim a deduction for a contribution of noncash property worth more than $500,000, you'll also need to attach the qualified appraisal to your return.
- In order to claim a tax deduction for nonprofit volunteer work, one has to charge said nonprofit for their time.
- Nonprofit software foundations can only pay volunteers if they have money in their treasury (e.g. from donations, merch sales, support services, subscriptions,)
  https://en.wikipedia.org/wiki/Business_models_for_open-source_software

**Comment by piamancini** [comment]
- Paypal - Done.

**Comment by BenJam** [comment]
+ Monzo, Tide

**Comment by auvipy** [comment]
what about Hyperwallet?

**Comment by Croydon** [comment]
Still relevant.

**Comment by lindner** [comment]
Another vote for transferwise

**Comment by mpl** [comment]
transferwise

**Comment by lsmith77** [comment]
I 2nd IBAN as very important.

I also suspect that Alipay would improve coverage across Asia.

**Comment by buger** [comment]
So far the most reliable provider which even works with countries like Russia or Ukraine, is https://paddle.com

**Comment by IzzySoft** [comment]
If you had SEPA, I'd consider joining. But Stripe & PayPal are not really privacy-friendly options (which is the reason why my LiberaPay account is currently "stalled" and unusable, since the SEPA option disappeared).

PS: It would also be a good idea to mention available payment processors in a place that's easy to find from the homepage :wink:

**Comment by lsmith77** [comment]
it seems to me like SEPA is now supported. at least on the Brussels OSS fiscal host

**Comment by znarf** [comment]
@lsmith77 it's not supported as automated payout. It's up to the host to read the manual instructions and to do the wire manually. It's ok for a few transactions but it doesn't scale right now.

**Comment by IzzySoft** [comment]
@znarf so if I want to manually withdraw 2, 3 times per year, that works without a "man in the middle"? No PayPal, Stripe, whatever?

**Comment by znarf** [comment]
@IzzySoft The hosts on Open Collective are separated independent entities, it's up to them to tell you what is ok or not. I do know that they currently prefer Paypal, because it's the only one that is automated.

**Comment by rugk** [comment]
BTW as for SEPA support, please see (or upvote :smile:) https://github.com/opencollective/opencollective/issues/1217, https://github.com/opencollective/opencollective/issues/1763 and https://github.com/opencollective/opencollective/issues/1763

**Comment by auvipy** [comment]
If you allow sending payoneer invoice withdrawal, then may people can send payoneer invoice to your invoice email and you could clear that with just US local bank to Payoneer without any fee at all [Not the shitty Payoneer global tranfer!!!] and Paypals - Hyperwallet and Zoom

**Comment by rugk** [comment]
this is an open poll, these bots these days… :roll_eyes:

**Comment by Betree** [comment]
@rugk we've configured this bot to never close issues. For us it's only a way to notice the ones that didn't receive any activity recently so we don't forget them.

**Comment by davidak** [comment]
...

**Comment by IzzySoft** [comment]
@Betree so it's better if we not "just ping to remove the ~stale" – but leave it there so it presses on your conscience? :thinking:

**Comment by bitcoinbrisbane** [comment]
> * Coinbase and BitPay are good centralized solutions / companies to handle payments in ETH, BTC, LTC and BCH.
> * I am working in [Request Network](https://request.network) team, we're creating a free decentralized payment requests network to get paid in any crypto-currency (including ETH, BTC, etc...).
>   The library is here https://github.com/RequestNetwork/

BTCPay Server is self hosted, 100% of crypto goes to the project.  Happy to set one up.
